drivers/network/dd/dc21x4/media.c

```diff
@@ -326,7 +326,7 @@ MediaInitDefaultMedia(
     _In_ PDC21X4_ADAPTER Adapter,
     _In_ ULONG MediaNumber)
 {
-    ULONG Csr14, MiiAdvertising, MiiControl, i;
+    ULONG Csr14, MiiAdvertising, i;
```

Comment:
Moving variables is optional.
If you move 1, then move 2.
Furthermore, you may move the other 2 as well.
```suggestion
    ULONG Csr14, i;
```

Reply:
I just moved the first one to make it more obvious to the reviewers that this variable isn't used before line 474. I won't move any other variable within this PR. No. I might even end with keeping the initial one where it was, **depending on @disean s personal preference where to place it.**
I don't want to mess with unrelated things in here.

Reply:
Yes it's obvious that since both variables are only used within this else + switch block below, they can be moved together as suggested without any problem.

Reply:
Alternatively, you could split this function into two.

Reply:
I would prefer to keep the changes simple and small. I did act on the first set of recommendations. Please rereview and try to answer my raised question for the reamaining = 0 assignment. Thx.